kernel/kernbench.py

```diff
@@ -38,6 +38,10 @@ class Kernbench(Test):
     and compile it as per the configuration file. Performance figures will be
     shown in the log file.
     """
+    process.system("cp /boot/config* /tmp", shell=True, sudo=True)
```

Comment:
it can be like as @harish-24 mentioned 

it can be applicbale for all rhel and rhel origin distro 
```
if detected_distro.name in ['centos', 'fedora', 'rhel']: 
   process.system("cp /boot/config* /tmp", shell=True, sudo=True)
   process.system("sed -i 's/^.*CONFIG_SYSTEM_TRUSTED_KEYS/#&/g' /boot/config*", shell=True, sudo=True)
   process.system("sed -i 's/^.*CONFIG_SYSTEM_TRUSTED_KEYRING/#&/g' /boot/config*", shell=True, sudo=True)
   process.system("sed -i 's/^.*CONFIG_MODULE_SIG_KEY/#&/g' /boot/config*", shell=True, sudo=True)
```

Reply:
please move this on setup section as well

Reply:
Thanks @harish-24 and @PraveenPenguin , I've changed the fix to be rhel or rhel origin distros specific.